benchmark/upload_scribe.py

```diff
@@ -91,23 +95,31 @@ def post_pytest_benchmarks(self, pytest_json):
         commit_info = pytest_json['commit_info']
         upload_time = int(time.time())
         messages = []
+        try:
+            import torch
+            torch_version = torch.__version__
+        except ImportError:
+            torch_version = None
```

Comment:
ought we eat the error here? If we're worried about torch not being available from the python running this upload script, could we just have the benchmark harness itself populate this info and pass it in through `pytest_json`?

Reply:
yea, this is lame.  i tried to pass additional info in the .json at one point and found it to be pretty limited, but i will take another look.